src/libs/ajax/leonardo/providers/LeoAppProvider.ts

```diff
@@ -0,0 +1,20 @@
+import { Ajax } from 'src/libs/ajax';
+import { ListAppResponse } from 'src/libs/ajax/leonardo/models/app-models';
+
+export interface LeoAppProvider {
+  listWithoutProject: (args: Record<string, string>, signal?: AbortSignal) => Promise<ListAppResponse[]>;
+  delete: (resourceName: string, appName: string, deleteDisk?: boolean, signal?: AbortSignal) => Promise<void>;
+  pause: (resourceName: string, appName: string, signal?: AbortSignal) => Promise<void>;
```

Comment:
I don't think either delete or pause have the right interface (see comment below on delete)

Reply:
fixed these up based on our convorsation.